app/services/forest_liana/schema_adapter.rb

```diff
@@ -193,6 +196,17 @@ def get_type_for(column)
       end
     end
 
+    def add_enum_values_if_is_enum(column_schema, column)
+      if column_schema[:type] == 'Enum'
+        column_schema[:enumValues] = []
+        @model.defined_enums[column.name].each do |name, value|
+          column_schema[:enumValues] << { name: name, value: value }
```

Comment:
Sometimes the "protocol" uses snake case and sometimes camel case.
Ex: `inverseOf` and `inverse_of`